dotcom-rendering/src/components/AppsLightboxImage.importable.tsx

```diff
@@ -70,9 +38,9 @@ export const AppsLightboxImage = ({
 			isMainMedia={isMainMedia}
 		/>
 	);
-	return hasLightbox ? (
+	return (
 		<button
-			onClick={onClick}
+			onClick={() => openLightboxForImageId(elementId)}
```

Comment:
Is it possible that you could click on this before the `AppsLightboxImageStore` has hydrated?

Reply:
Yes, possibly. I don't know how quick you'd have to be though.

The fix would be to have the `store` `Island` as `critical`, I think.